### ISSUE TITLE
Getting lesser accuracy on omniglot dataset than mentioned in the original paper.

### ISSUE THREAD

**Issue by devil10** [AntreasAntoniou/HowToTrainYourMAMLPytorch#20]
When trained the maml++ model on omniglot dataset (1 shot 20 way) with seed 0, I get 96.59 accuracy on test data which is more than 1% less than the accuracy claimed in the original paper. What might be the issue ?

**Comment by AntreasAntoniou** [comment]
The paper results represent the mean and standard deviation of the test accuracy performance over 3 seeds. You might have to do the same to replicate the results.

**Comment by devil10** [comment]
I get 96.65 as my mean across three seeds and 0.0587 as my std across three seeds. 
Pytorch Version -> 1.0.1.post2
Cuda Version -> 9.0.176
Cudnn Version -> 7402

**Comment by AntreasAntoniou** [comment]
I wonder if there are any minor seed differences between different PyTorch
versions. I'll need to have a more thorough look.
